exonum-java-binding/core/rust/src/runtime/java_runtime_factory.rs

```diff
@@ -32,15 +32,15 @@ const LOAD_ARTIFACT_SIGNATURE: &str = "(Ljava/lang/String;)Ljava/lang/String;";
 const CREATE_SERVICE_SIGNATURE: &str =
     "(Ljava/lang/String;)Lcom/exonum/binding/core/service/adapters/UserServiceAdapter;";
 
-/// Controls JVM and java service.
+/// Controls JVM and java runtime.
 #[allow(dead_code)]
 #[derive(Clone)]
-pub struct JavaServiceRuntime {
+pub struct JavaRuntimeFactory {
```

Comment:
Better names? Maybe JavaSystemRuntime? @vitvakatu, will it survive your changes?
 Also, I don't see service_runtime to be used at all after initialization 💭

Reply:
I haven't decided yet, but it looks like this structure will be removed (it can be replaced with a few functions, as there is no need in saving any state). Though I think `JavaRuntimeFactory` is a good name for a some thing that creates JavaRuntimeProxy.

Reply:
Ok, let's keep as is, maybe with a todo referencing your task.